Improve/accessibility

bmlancien opened the pull request:
Closes #144

nesnoj commented:
Thank you for spotting and this PR @bmlancien! Please go ahead fixing the most important (!) things and I can review as soon as it is ready.

bmlancien commented:
Alright @nesnoj. 

In this PR I mostly did the following:

- improved keyboard navigation (now the tooltip contents should be visible and selecting a scenario should be possible)
- added missing aria attributes and aria label descriptions as well as improved HTML5 semantics (useful for navigating the app with screen readers)
- enabled screen reader description of charts (see [documentation](https://echarts.apache.org/en/option.html#aria.enabled)). I read that the configurated language will be used to build the description, and I don't think it's the case now, so when it was possible I added a description in German made of the chart's title and legend data.
- corrected color contrast

I didn't work on making the map more understandable, as this is too much of a process to do it properly. I didn't see any accessibility option directly on the MapLibre page, only [this](https://github.com/mapbox/mapbox-gl-accessibility/).

There are also some areas where I probably missed things or didn't implement the correct way. For example I was technically not able to link the slider labels to the sliders for the screen readers. I'm also not very sure of how easy it is no navigate with a screen reader on a web app that actually consists of only one page but have different views toggled on and off... But after that PR the accessibility score should - hopefully - be a bit better.

For future tools I would recommend to focus more on the accessibility issue, especially accessibility in data visualization. I will try to gather more knowledge in that field as it is very relevant to the tools developed by the institute. I would also find it interesting to test future tools with users with disabilities to see how good/bad the apps are.

nesnoj reviewed (approved):
Thx @bmlancien for these improvements!
